runtime/src/main/jni/ModuleInternal.cpp

```diff
@@ -155,6 +157,8 @@ void ModuleInternal::RequireCallbackImpl(const v8::FunctionCallbackInfo<v8::Valu
     }
 
     string moduleName = ArgConverter::ConvertToString(args[0].As<String>());
+    string frameName("RequireCallback " + moduleName);
```

Comment:
Can this be inlined with the constructor on the following line? Or rename the frameName variable accordingly so that it becomes clear the `RequireCallback moduleName` string isn't part of the require logic